web/static/css/app.css

```diff
@@ -0,0 +1,38 @@
+ul {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 55px;
+}
+ul {
```

Comment:
Let's combine some of these duplicate styles. Might as well just have one ul style with all the content from both